cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -197,18 +202,39 @@ class ARROW_DS_EXPORT RadosParquetFileFormat : public ParquetFileFormat {
   std::shared_ptr<DirectObjectAccess> doa_;
 };
 
+/// \brief Serialize scan request to a bufferlist.
+/// \param[in] options Scan options.
+/// \param[in] file_size File size.
```

Comment:
Here, since this is on the client, we need to say, "The size of the file fragment"

Reply:
Done.